Library/Homebrew/tap.rb

```diff
@@ -171,17 +171,34 @@ def core_tap?
   # @option options [Boolean] :quiet If set, suppress all output.
   def install(options = {})
     require "descriptions"
-    raise TapAlreadyTappedError, name if installed?
-    clear_cache
 
+    installed = installed?
```

Comment:
Is it worthwhile to cache that value locally for the following code instead of using `installed?` directly.